### PR TITLE
Update RHS compat for 0.3.8

### PR DESCRIPTION
**Pull request by PabstMirror**

### CODE REVIEW

File: optionals/compat_rhs_usf3/CfgWeapons.hpp [hunk 1]
```diff
@@ -6,74 +6,56 @@ class CfgWeapons
...
+    class rhs_weap_m16a4: rhs_weap_m4_Base {
         ACE_barrelTwist=177.8;
         ACE_barrelLength=508.0;
     };
-    class rhs_M249_base;
-    class rhs_weap_m249_pip: rhs_M249_base
-    {
-        ACE_barrelTwist=177.8;
-        ACE_barrelLength=414.02;
-    };
```
Comment (ulteq) [inline optionals/compat_rhs_usf3/CfgWeapons.hpp]: Did RHS remove/rename their M249?

**Comment by esteldunedain** [comment]
What happened to the 249?

**Comment by PabstMirror** [comment]
New M249s are from HLC, so they actually have ace_xxx values on them

**Comment by esteldunedain** [comment]
> New M249s are from HLC

I'm glad they went that route.

**Comment by ulteq** [comment]
> New M249s are from HLC, so they actually have ace_xxx values on them

Cool!

**Comment by ViperMaul** [comment]
This is good to see.

**Pull request merged**